ipn/local.go

```diff
@@ -63,8 +63,8 @@ type LocalBackend struct {
 	stateKey     StateKey
 	prefs        *Prefs
 	state        State
-	hiCache      *tailcfg.Hostinfo
-	netMapCache  *controlclient.NetworkMap
+	hostinfo     *tailcfg.Hostinfo
```

Comment:
document whether these two pointed-to values are mutable or not once set under mu?